test/DynamoCoreTests/SerializationTests.cs

```diff
@@ -52,6 +81,18 @@ public void TearDown()
             ExecutionEvents.GraphPostExecution -= ExecutionEvents_GraphPostExecution;
         }
 
+        private void DeleteJsonFilesFromTemp(List<string> files)
+        {
+            foreach (var file in files)
+            {
+                //Delete files if they exist
+                if (File.Exists(file))
```

Comment:
This check isn't necessary (and the entire function may not be necessary if Directory.Delete works).  If the file didn't exist, it wouldn't be in the list.  If it did exist when the list was generated, but doesn't now, then it's still safe to call `File.Delete` on it, as deleting a non-existent file doesn't throw an exception.

Reply:
I agree. If I am using `Directory.Delete`, I can remove this method entirely.